Assets/InputSystem/StarterAssetsInputs.cs

```diff
@@ -57,14 +58,21 @@ public void OnSprint(InputValue value)
 		public void OnRaiseLantern(InputValue value)
 		{
 			RaiseLanternInput(value.isPressed);
-    }
+		}
 		public void OnInteract(InputValue value)
 		{
-			InteractInput(value.isPressed);
+			InteractInput(value.isPressed);	
 		}
-#endif
-
+		public void OnPressingQ(InputValue value)
+        {
+			PressingQInput(value.isPressed);
```

Comment:
I'm not sure i like the fact that "OnPressingQ" is a different function then from Interact. In the future we might want to have two keys binded to a same action that we can then differ inside the function.